ci-operator/step-registry/openshift/e2e/openstack/csi/OWNERS

```diff
@@ -1,4 +1,13 @@
 approvers:
```

Comment:
Please leave `openstack-approvers` and `openstack-reviewers`. We'd like to be able to make modifications to the job, for example moving it from MOC to Vexxhost, without depending on the storage team for reviews.

Reply:
done!